eggdrop.conf

```diff
@@ -615,6 +615,19 @@ die "Please make sure you edit your config file completely."
 set mod-path "modules/"
 
 
+#### PBKDF2 MODULE ####
+#loadmodule pbkdf2
+
+# Encryption method used. openssl list -digest-algorithms
+#set pbkdf2-method "SHA512"
+
+# Enable re-encoding of password if pbkdf2-method and / or pbkdf2-rounds change.
+#set pbkdf2-re-encode 0
+
+# Number of rounds.
+#set pbkdf2-rounds 5000
```

Comment:
Assume I'm an idiot. What is rounds and why is 5000 a good default? Just asking for an improved comment for the user.

Reply:
the pbkdf2 method adds CPU penalty to an attacker by not only encryptiong a pwd once, but n-times. The n is called rounds or iteration count.

For example glibc uses rounds=5000

https://github.com/lattera/glibc/blob/master/crypt/sha256-crypt.c#L88

The more rounds, the more CPU you put use to crack or to encode a password. You can always go higher, but remember: eggdrop is single threaded. Any passowrd encoding in eggdrop will not only slow the encode down, but block eggdrop as long as it takes. Therefore i also implemented a small debug log showing how many millisecs it took.

Reply:
I hope,Geo jumps in and does write some nice documentation for the soon to be finished (?) pbkdf2 module thereby explaining questions like that.

Reply:
Oh, and i tried to explain rounds and its effect in the Text in this PR. I copy the text here also:

    Add benchmark debug for pbkdf2, so one can see how rounds effects the time needed to hash a password. My developer system for example shows 5ms for default values (sha512 with 5000 rounds):

.chpass testuser 123456  
[10:35:15] pbkdf2 method SHA512 rounds 5000, user 5.284ms sys 0.000ms